### DISCUSSION TITLE
Free tier does not reset

### DISCUSSION THREAD

**Discussion by technote-space** [community/community#26524] [Actions]
<p>I received a notification that I’ve used 75% of my spending limit for GitHub Actions.</p>
<p>&gt; To ensure your service isn’t disrupted, make sure you increase your monthly spending limit.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/3/32ee2fc478102e170262acc2ef86a777716908d0.png" data-download-href="/uploads/short-url/7gybwL3OAZRhLTcelCNcsGKQsGQ.png?dl=1" title="billing.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/39912269/181098153-da9f050b-a82f-43c2-aa2e-5f420ba3b49b.png" alt="billing.png" data-base62-sha1="7gybwL3OAZRhLTcelCNcsGKQsGQ" width="503" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/32ee2fc478102e170262acc2ef86a777716908d0_2_503x500.png, https://aws1.discourse-cdn.com/github/original/2X/3/32ee2fc478102e170262acc2ef86a777716908d0.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/3/32ee2fc478102e170262acc2ef86a777716908d0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/32ee2fc478102e170262acc2ef86a777716908d0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">billing.png</span><span class="informations">681×676 41.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>It looks like my free tier has not been reset.</p>
<p>Do I need to update my spending limits to continue using GitHub Actions in my private repository?</p>
<p>Payment history:</p>
<p><img src="https://user-images.githubusercontent.com/39912269/181098154-493f39e9-8c7b-4c60-aab7-dbce07a7d9ac.png" alt="history.png" data-base62-sha1="mpYatuqMBOGApZLhfcbVRNaz5fQ" width="290" height="209"></p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/technote-space">@technote-space</a>,</p>
<p>Yes, please follow the highlighted message to update spending limit.</p>
<p><span>For accounts that have reached the included usage for the current billing period for GitHub Actions or GitHub Packages, both GitHub Actions and GitHub Packages will be disabled to prevent any further overages.</span></p>
<p><span>Please check the billing doc <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions#about-spending-limits" rel="nofollow noopener">here</a> for the details.</span></p>
<p><span>Thanks.</span></p>

**Comment by technote-space** [comment]
<p><a class="mention" href="/weide-zhou">@weide-zhou</a> </p>
<p>Thank you for your reply.</p>
<p>I understood that “-7 days” is the correct behavior and the free tier will not reset.</p>

**Comment by technote-space** [comment]
<p><a class="mention" href="/weide-zhou">@weide-zhou</a> </p>
<p>I read the page.</p>
<p><a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions" rel="nofollow noopener">https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions</a></p>
<p>&gt; Minutes reset every month, while storage usage does not.</p>
<p>Does this not apply to all users?</p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/technote-space">@technote-space</a> ,</p>
<p>It applies to all users. For private repositories, each GitHub account receives a certain amount of free minutes and storage(storage not reset every month), the spending minutes means you expand the default $0 with the ‘overages’, and you have used more than 75% of it. Please refer to the <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/managing-your-spending-limit-for-github-actions#about-spending-limits-for-github-actions" rel="nofollow noopener">About spending limits for GitHub Actions </a>for more details.</p>
<p>It’s recommended to update the spending limit since your action service could be discrupted by all spending limit consumed. Or course, you can also decrese the service use and make sure it will not consume all spending limit minutes before free minutes reset.</p>

**Comment by technote-space** [comment, marked as answer]
<p>After downgrading to free and switching to yearly billing, this strange issue disappeared.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/1bbc648db6a3a4a198101d59bed27112221d04f6.png" data-download-href="/uploads/short-url/3XmwkdfIkZJ5JhXoufMPi7v0C1g.png?dl=1" title="2020-04-15 13.20.13.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/39912269/181098160-de8b9e7b-2d20-4181-9a5d-3db5e2cc6294.png" alt="2020-04-15 13.20.13.png" data-base62-sha1="3XmwkdfIkZJ5JhXoufMPi7v0C1g" width="670" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/1bbc648db6a3a4a198101d59bed27112221d04f6_2_670x500.png, https://aws1.discourse-cdn.com/github/original/2X/1/1bbc648db6a3a4a198101d59bed27112221d04f6.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/1/1bbc648db6a3a4a198101d59bed27112221d04f6.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/1bbc648db6a3a4a198101d59bed27112221d04f6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2020-04-15 13.20.13.png</span><span class="informations">709×529 38.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
